fixtures/cameo/zenit-z120.json

```diff
@@ -0,0 +1,422 @@
+{
+  "$schema": "https://raw.githubusercontent.com/OpenLightingProject/open-fixture-library/master/schemas/fixture.json",
+  "name": "Zenit Z120",
+  "shortName": "Z120",
+  "categories": ["Color Changer"],
+  "meta": {
+    "authors": ["EP", "Ken Harris", "Luke Nelson"],
+    "createDate": "2024-01-06",
+    "lastModifyDate": "2024-01-06"
+  },
+  "links": {
```

Comment:
It would be great to have links to the product page and a video, if possible. Not merge-blocking though.

Reply:
The Zenit Z120 is apparently succeeded by the Zenit Z120 G2 (which is slightly different: see #3484), and Cameo seems to be one of those companies which scrub any evidence of earlier products from their website.

They do still have a Z120 video on their YouTube channel, though: https://www.youtube.com/watch?v=q3aCVNj3t7M

Based on the video description, it looks like the old URL was http://www.cameolight.com/en/zenit-series/zenit-z-120-professional-zoom-par-light-with-ip65/ but that's just an error page now.  Archive.org has captured the ZENIT SERIES page that links to it, but not the Z120 product page itself: https://web.archive.org/web/20160414043113/http://www.cameolight.com/en/zenit-series/

Reply:
Thanks for checking! Then let's include the video only.